src/types/map.go

```diff
@@ -410,6 +410,10 @@ func (m Map) LazyArrayElemType() Map {
 
 	mm := make(map[string]struct{}, m.Len())
 	for typ := range m.m {
+		if typ == "null" {
```

Comment:
As a crutch OK :(

But we should add a TODO comment that there should be an inspection here that only iterators can be unpacked.